package NArcProto;

enum EArchiveType {
    AT_FLAT = 0;
    AT_MULTIPART = 1;
}

enum EArchiveVersion {
    ARCVERSION = 1;
    ARC_COMPRESSED_EXT_INFO = 2;
}
